Formula/n/netatalk.rb

```diff
@@ -57,22 +55,23 @@ def install
     inreplace "distrib/initscripts/macos.netatalk.plist.in", "@bindir@", opt_bin
     inreplace "distrib/initscripts/macos.netatalk.plist.in", "@sbindir@", opt_sbin
     inreplace "distrib/initscripts/systemd.netatalk.service.in", "@sbindir@", opt_sbin
-    inreplace "config/meson.build", "cups_libdir / 'cups/backend'", "'#{libexec}/cups/backend'"
     bdb5_rpath = rpath(target: Formula["berkeley-db@5"].opt_lib)
     ENV.append "LDFLAGS", "-Wl,-rpath,#{bdb5_rpath}" if OS.linux?
     args = [
       "-Dwith-afpstats=false",
       "-Dwith-appletalk=#{OS.linux?}", # macOS doesn't have an AppleTalk stack
       "-Dwith-bdb-path=#{Formula["berkeley-db@5"].opt_prefix}",
-      "-Dwith-docbook-path=#{Formula["docbook-xsl"].opt_prefix}/docbook-xsl",
+      "-Dwith-cups-libdir-path=#{libexec}",
+      "-Dwith-cups-pap-backend=true",
```

Comment:
We would only want to set this to `true` for Linux, since we don't use CUPS with netatalk on macOS.